src/languages/typescript.coffee

```diff
@@ -18,4 +18,6 @@ module.exports = {
     "ts"
   ]
 
+  defaultBeautifier: "TypeScript Formatter"
```

Comment:
👍 This was the previous default beautifier.

Reply:
Yep, the build failed originally because it used prettier for the tests.